app/js/handlebarsScoreTable.ts

```diff
@@ -64,7 +64,7 @@ function printFilteredResultsToScreen(HBTemplate: string, scoresDataArray: Array
  * @param event is the event fired off by the function
  */
 function createObjectFromParentElement(event: Event) {
-    let parentElement = (event.target as HTMLElement).parentElement;
+    let parentElement = (event.target as HTMLElement);
```

Comment:
you are no longer getting the parent of the target element - therefore the function should be renamed to `createObjectFromElement` and the variable should be renamed to `element`